src/api/java/com/minecolonies/api/colony/IColony.java

```diff
@@ -475,4 +478,17 @@ default List<BlockPos> getWayPoints(@NotNull BlockPos position, @NotNull BlockPo
      * @return the current day progress of the colony.
      */
     int getDay();
+
+    /**
+     * Get the quest manager of the colony.
+     * @return the quest manager.
+     */
+    IQuestManager getQuestManager();
+
+    /**
+     * Get citizen from colony.
+     * @param id the id of the cit.
+     * @return the cit.
+     */
+    ICitizen getCitizen(int id);
```

Comment:
Its because of the client/server separation. citizen manager doesnt exist on the client. Would require quite the large rework to make sure we dont expose methods to the client that are unusable.